NCMP_for_2-1-1_and_2-2-1/WHO_code_test.R

```diff
@@ -0,0 +1,97 @@
+###############################################################################################
+#
+# The user must set the working directory to match the directory of this R script
+# setwd() # uncomment this line to set the directory of the downloded script,, + csv files, keeping the folder with the data one directory above
+#
+#
+#NB: The WHO function needs to be run on each year of data separately - so 1 csv file per year
+###############################################################################################
+
+# install the package and load it in your R session:
+#install.packages("anthro",dependencies=TRUE, type="win.binary") #only need to do this once
+library(anthro) #need to run every time you start a new session, if you want to use the package
+
+
+# reading the data from 2018-19 download of the zipped file from NHS Digital, after extracting
+# assuming the data csv is one level up from the current working directory
+NCMP_data <- read.csv("../ncmp_1819_final_non_disclosive_published.csv")
```

Comment:
You could dispense of the need to setwd  (which is an extra step that can go wrong), and instead just have the full filepath as a configuration